config/routes.rb

```diff
@@ -343,16 +343,6 @@
 
   namespace :api do
     get '/', to: redirect('/help/api', status: 302)
-    namespace :internal do
-      namespace :v1 do
-        get '/users/:id/permissions' => 'permissions#index'
-        get '/competitions/:competition_id' => 'competitions#show'
-        get '/competitions/:competition_id/qualifications' => 'competitions#qualifications'
-        post '/users/competitor-info' => 'users#competitor_info'
-        post '/mailers/registration' => 'mailers#registration'
-        post '/payment/init_stripe' => 'payment#init_stripe'
-      end
-    end
```

Comment:
Where did these routes go?

Reply:
well, I started out this PR by adding the results route as an internal route. Then I noticed that we had all these dead routes under the internal namespace that we forgot to delete in v1 cleanup